pontoon/base/urls.py

```diff
@@ -55,9 +55,15 @@
         name='pontoon.project.contributors'),
 
     # AJAX: Get project details
-    url(r'^projects/(?P<slug>[\w-]+)/details/$', views.get_project_details,
+    url(r'^projects/(?P<slug>[\w-]+)/details/$',
+        views.get_project_details,
         name='pontoon.project.details'),
 
+    # AJAX: Toogle user profile attribute
+    url(r'^toggle-user-profile-attribute/(?P<attribute>[\w-]+)/$',
```

Comment:
This URL is too specific (the old one was too). If we follow this pattern we'd have to create a new URL for every individual action we want to do even if many of them alter the same resource. 

Ideally we try to be RESTful and have the URL point to the user's profile (not necessarily the same URL as their profile page, can be an API-specific endpoint like `/api/v1/user/{id}/`), and the POST arguments are the attribute names that we compare to the whitelist.

Even more ideally we use something like Django REST Framework to implement this API but that's way too out of scope for this PR to consider. But perhaps we can inch closer incrementally in the meantime?